### PR TITLE
SSR - Registration Activation: Adding view and override 'back to sign in' link

### PR DESCRIPTION
**Pull request by justinabrokwah-okta**
## Description:
- Added new view based on [figma](https://www.figma.com/file/fZGJS3FV0RpfKImY8CKwij/Magic-Link?node-id=1978%3A58413) (SSR Flows - Section 6), to allow user to resend activation email and restart email verification
- Allow for overriding back to sign in link to redirect to specific app instead of default app

## PR Checklist

- [ ] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

- Did you verify the change by running downstream monolith artifacts? (YES | NO | UNSURE)

### Screenshot/Video:
- [OKTA-489026 screenshots](https://okta.box.com/s/ma0x5w11xp1picx1ofnuhep5bfpu9gcq)

### Issue:

- [OKTA-489026](https://oktainc.atlassian.net/browse/OKTA-489026)

### CODE REVIEW

File: src/v2/view-builder/views/activation/EMLActivationSessionExpiredView.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+const Footer = BaseFooter.extend({
+  hasBackToSignInLink: false, // override default 'back to sign in' link to replace w/ our own URL
+  links: function() {
+    // console.log(this.options.appState);
+    // get 'verify with something else' link
+    const verifyWithLink = getSwitchAuthenticatorLink(this.options.appState);
+
+    // generate back to sign in link
+    // can set URL in value:
+    // this.options.settings.set('baseUrl', 'https://www.okta.com/');
```
[the remark this answers is not in the document]
Comment (jakebacher2-okta) [reply src/v2/view-builder/views/activation/EMLActivationSessionExpiredView.js:28]: I'm curious, why would we want to overwrite this? Wouldn't we always want the end user to be able to go back to the original sign-in page?

**Comment by jakebacher2-okta** [comment]
@justinabrokwah-okta just looking at the screenshots. Are we planning to remove the yellow warning callout `Haven't received an email? Send again` with your change. Seems redundant with this change as there will now be 2 buttons to initiate the same action of resending the email. 

CC: @bhagyaprabhakar-okta

File: packages/@okta/i18n/src/properties/login.properties [hunk 2]
```diff
@@ -1416,6 +1416,10 @@ oie.activation.request.email.title.suspended = Account suspended
 oie.activation.request.email.title.submitted = Request submitted
 oie.activation.request.email.button= Request activation email
 
+idx.eml.activation.session.expired.title = Your session has expired
+idx.eml.activation.session.expired.instructions = Send <span class="strong">{0}</span> a new link
```
[the remark this answers is not in the document]
Comment (justinabrokwah-okta) [reply packages/@okta/i18n/src/properties/login.properties:1420]: Good catch, it's also needed here I'm pretty sure - will update

File: src/v2/view-builder/views/activation/EMLActivationSessionExpiredView.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (hanyuanzhang-okta) [reply src/v2/view-builder/views/activation/EMLActivationSessionExpiredView.js:27]: Is `this.options.settings` part of response from okta-core? If yes can you list what are available (e.g. top elements `remediation`, `user`, `app`, `cancel`)?

[the remark this answers is not in the document]
Comment (justinabrokwah-okta) [reply src/v2/view-builder/views/activation/EMLActivationSessionExpiredView.js:27]: @hanyuanzhang-okta No, `settings` is not a part of the response, looks like it's dynamically constructed [here](https://github.com/okta/okta-signin-widget/blob/8b536fa28361b6362583da361a4bb7ca4a6f5adf/src/models/Settings.ts#L371) and contains information that could be useful for all views.

For the other items you indicated, those come through in the response from okta-core and are found in `this.options.appState`. I used a mock json for request-activation-email as the basis for the mock json in this PR (`email-registration-activation-session-expired.json`) - i know our backend isn't completed so this json can still be changed

[the remark this answers is not in the document]
Comment (hanyuanzhang-okta) [reply src/v2/view-builder/views/activation/EMLActivationSessionExpiredView.js:27]: Thank you @justinabrokwah-okta As long as the code here has access to those fields I can focus on building a `redirectUri` that points to the app and include it in the `IdxResponse`. Not sure if it is feasible yet but at least the SIW is not a blocker 💯

File: src/v2/view-builder/views/activation/EMLActivationSessionExpiredView.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+  save() {
+    return loc('idx.eml.activation.session.expired.button.label', 'login');
+  },
+});
+
+const Footer = BaseFooter.extend({
+  hasBackToSignInLink: false, // override default 'back to sign in' link to replace w/ our own URL
+  links: function() {
+    // console.log(this.options.appState);
```
[the remark this answers is not in the document]
Comment (justinabrokwah-okta) [reply src/v2/view-builder/views/activation/EMLActivationSessionExpiredView.js]: Removed this line

File: packages/@okta/i18n/src/properties/login.properties (hunk 2 above)
[the remark this answers is not in the document]
Comment (justinabrokwah-okta) [reply packages/@okta/i18n/src/properties/login.properties:1421]: @jakebacher2-okta added the comment explaining the string parameter

**Comment by justinabrokwah-okta** [comment]
Closing this PR for now as the UI work has been de-prioritized

**Pull request closed without merging**